src/main/resources/anniv5Resources/localization/eng/magnetizepack/Cardstrings.json

```
@@ -0,0 +1,67 @@
+{
+  "${ModID}:Attraction": {
+    "NAME": "Attraction",
+    "DESCRIPTION": "Draw !M! cards and ${ModID}:Magnetize them. NL Add a *Fuzz to your hand."
+  },
+  "${ModID}:CoilBurst": {
+    "NAME": "Coil Burst",
+    "DESCRIPTION": "Deal !D! damage. NL Add 2 *Fuzz to your hand."
+  },
+  "${ModID}:ElectronFlow": {
+    "NAME": "Electron Flow",
+    "DESCRIPTION": "Discard any number of cards. Gain !M! *Polarity for each card discarded. NL Add a *Fuzz to your hand.",
+    "EXTENDED_DESCRIPTION": ["Discard"]
+  },
+  "${ModID}:FaradayField": {
+    "NAME": "Faraday Field",
+    "DESCRIPTION": "Deal !D! damage to ALL enemies. NL ${ModID}:Magnetize your entire hand."
+  },
+  "${ModID}:Ferroshield": {
+    "NAME": "Ferroshield",
+    "DESCRIPTION": "Gain !B! Block. NL ${ModID}:Magnetize !M! random cards in your draw pile."
+  },
+  "${ModID}:Fuzz": {
+    "NAME": "Fuzz",
+    "DESCRIPTION": "Can only be played if ${ModID}:Magnetized. NL Exhaust.",
+    "EXTENDED_DESCRIPTION": ["That card isn't Magnetized!"]
+  },
+  "${ModID}:Hyperpolarity": {
+    "NAME": "Hyperpolarity",
+    "DESCRIPTION": "The next ${ModID}:Magnetized card you play this turn is played twice. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "The next ${ModID}:Magnetized card you play this turn is played twice."
+  },
+  "${ModID}:InduceCurrent": {
+    "NAME": "Induce Current",
+    "DESCRIPTION": "When you ${ModID}:Magnetize a card, deal !M! damage to a random enemy."
+  },
+  "${ModID}:MagneticDomain": {
+    "NAME": "Magnetic Domain",
+    "DESCRIPTION": "${ModID}:Magnetize and discard a card. Play a ${ModID}:Magnetized card in your discard pile. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "${ModID}:Magnetize and discard a card. Play a ${ModID}:Magnetized card in your discard pile.",
+    "EXTENDED_DESCRIPTION": ["Magnetize and discard", "play"]
+  },
+  "${ModID}:OhmicPush": {
+    "NAME": "Ohmic Push",
+    "DESCRIPTION": "Apply !${ModID}:m2! ${ModID}:Distortion and !M! Weak. NL Add a *Fuzz to your hand."
+  },
+  "${ModID}:PerfectedAlignment": {
+    "NAME": "Perfected Alignment",
+    "DESCRIPTION": "Deal !D! damage. NL Increase by !M! for ALL your ${ModID}:Magnetized cards."
+  },
+  "${ModID}:RepulsionField": {
+    "NAME": "Repulsion Field",
+    "DESCRIPTION": "Whenever a ${ModID}:Magnetized card is discarded or Exhausted, gain !M! Block."
+  },
+  "${ModID}:ElectromagneticSynergy": {
+    "NAME": "Electromagnetic Synergy",
+    "DESCRIPTION": "${ModID}:Magnetized cards deal !M! more damage and gain !M! more Block this turn."
+  },
+  "${ModID}:Superconductance": {
+    "NAME": "Superconductance",
+    "DESCRIPTION": "When you ${ModID}:Magnetize a card this turn, reduce its cost by !M! this turn until played."
```

```suggestion
    "DESCRIPTION": "When you ${ModID}:Magnetize a card this turn, reduce its cost by !M! this turn."
```
Mummified hand doesn't say until played either, even though it kinda is.

---

I'd like to make it clear if I have room. I know Mummified Hand doesn't specify but I kinda consider that... a mistake. Especially since without that addition Superconductance might look like it's built for infinites.

---

Personally I prefer sticking close to base game wording since that's already understood, however if you feel strongly about it, feel free to keep it.